themes/default/template/picture.tpl

```diff
@@ -65,9 +65,41 @@ function changeImgSrc(url,typeSave,typeMap)
 	</a>
 {/if}{/strip}
 {strip}{if isset($current.U_DOWNLOAD)}
-	<a href="{$current.U_DOWNLOAD}" title="{'Download this file'|@translate}" class="pwg-state-default pwg-button" rel="nofollow">
+	<a id="downloadSwitchLink" href="{$current.U_DOWNLOAD}" title="{'Download this file'|@translate}" class="pwg-state-default pwg-button" rel="nofollow">
 		<span class="pwg-icon pwg-icon-save"></span><span class="pwg-button-text">{'Download'|@translate}</span>
 	</a>
+
+{if !empty($current.formats)}
+{footer_script require='jquery'}{literal}
+jQuery().ready(function() {
+  jQuery("#downloadSwitchLink").removeAttr("href");
+
+  jQuery("#downloadSwitchLink").click(function() {
```

Comment:
Try using the build-in switchbox.js : `(window.SwitchBox=window.SwitchBox||[]).push("#downloadSwitchLink", "#downloadSwitchBox");`